bundown/bunfig.toml

```diff
@@ -0,0 +1,6 @@
+telemetry = false
+
+preload = ["./src/plugin/register.ts"]
+
+[install.lockfile]
+print = "yarn"
```

Comment:
I also don't think we should use the `yarn.lock` file, if we do want to commit a lockfile I prefer `bun.lockb` - you can optionally configure a yarn-style diff like this in your personal `.gitconfig` to see it in git diffs:

```.gitconfig
[diff "lockb"]
  textconv = bun
  binary = true
```
Overall though, this falls into the set of things I'd like to just keep simple and ideally not even have to use, not having a lockfile to begin with was intentional.

Reply:
I don't think not having a lockfile is the right way to go, we need to be able to see when/how our dependencies change at the very least